org.osgi.util.feature/src/org/osgi/util/feature/Feature.java

```diff
@@ -35,35 +37,63 @@
     String getName();
 
     /**
-     * Get the description.
-     * @return The description.
-     */
+	 * Get the categories.
+	 * 
+	 * @return The categories.
+	 */
+	Collection<String> getCategories();
+
+	/**
+	 * Get the copyright.
+	 * 
+	 * @return The copyright.
+	 */
+	String getCopyright();
+
+	/**
+	 * Get the description.
+	 * 
+	 * @return The description.
+	 */
     String getDescription();
 
+	/**
+	 * Get the documentation URL.
+	 * 
+	 * @return The documentation URL.
+	 */
+	URI getDocURL();
```

Comment:
This should probably just be `String` and let the user decide whether to parse into a URI or not.